src/main/java/edu/tamu/scholars/middleware/export/service/CsvExporter.java

```diff
@@ -150,4 +165,25 @@ private String serialize(String value) {
             : value;
     }
 
+    private String capitalize(String text) {
+        if (text == null || text.isEmpty()) {
+            return text;
+        }
+        return text.substring(0, 1).toUpperCase() + text.substring(1);
+    }
+
+    private String formatPersonType(String type) {
+        if (type == null || type.isEmpty()) {
+            return type;
+        }
+        String mapped = config.getPersonTypeMapping().get(type);
+        if (!mapped.equals(type)) return mapped;
```

Comment:
```suggestion
        if (config.getPersonTypeMapping().containsKey(type)) return config.getPersonTypeMapping().get(type);
```

The `type` is the key and the return of `.get(type)` is the value at the key.
This should not be comparing the map key to the map value.

Instead, check if the key exists and if it does, then return the value as-is.